executor/chunk_size_control_test.go

```diff
@@ -121,8 +118,7 @@ type testChunkSizeControlSuite struct {
 	m map[string]*testChunkSizeControlKit
 }
 
-func (s *testChunkSizeControlSuite) SetUpSuite(c *C) {
-	c.Skip("not stable because coprocessor may result in goroutine leak")
+func (s *testChunkSizeControlSuite) SetUpSuite(t *testing.T) {
```

Comment:
I'd prefer a `createChunkSizeControlSuite(t *testing.T) *ChunkSizeControlSuite` method.

Reply:
I'm pushing a patch on this now...